stock_picking_delivery_slip_a5/i18n/es.po

```diff
@@ -20,12 +20,12 @@ msgstr ""
 #. module: stock_picking_delivery_slip_a5
 #: model:ir.actions.report,print_report_name:stock_picking_delivery_slip_a5.action_report_stock_picking_document_scr_a5
 msgid "'Delivery slip A5 - %s - %s' % (object.partner_id.name or '', object.name)"
-msgstr ""
+msgstr "'Albarán A5 - %s - %s' % (object.partner_id.name or '', object.name)"
 
 #. module: stock_picking_delivery_slip_a5
 #: model_terms:ir.ui.view,arch_db:stock_picking_delivery_slip_a5.report_stock_picking_document_scr_a5
 msgid "<span><b>Contact:</b></span><br/>"
-msgstr "Contacto"
+msgstr "<span style=\"font-weight: bold;\">Contacto:</span><br/>"
```

Comment:
I'm not sure why translated code is coded different than original code (`<b>` vs style font), this could lead to some confusion when we'll go back to this code, but it's actually irrelevant.
And not sure about "Contact" legend is bold, but "VAT" not (visual incoherence). Anyway, this issue does not belong to this PR (not a translation issue).
We'll, I'll merge it